src/pages/SettingsPage.js

```diff
@@ -72,6 +72,22 @@ const Settings = ({ toRealmExplorer, currentUserPasswordHash, requestStorageWrit
     if (!ValidUrl.isWebUri(newSyncURL)) ToastAndroid.show('Not a valid URL', ToastAndroid.LONG);
     else setState({ ...state, modalKey: '', syncURL: newSyncURL });
   };
+
+  const onCheckConnection = () => {
+    //  Hash the password.
+    const AUTH_ENDPOINT = '/sync/v3/site';
+    const authURL = `${syncURL}${AUTH_ENDPOINT}`;
+    const syncSiteName = UIDatabase.getSetting(SETTINGS_KEYS.SYNC_SITE_NAME);
+    const syncAuthenticator = new SyncAuthenticator(AppSettings);
+    syncAuthenticator
+      .authenticate(authURL, syncSiteName, null, currentUserPasswordHash)
+      .then(() => {
+        ToastAndroid.show('Connection OK', ToastAndroid.LONG);
```

Comment:
Can you add this string into the localized strings, please? For now, just in the `en` part of `syncStrings` would be good!

Reply:
This is fixed by this: 6911d06